app/src/androidTest/java/ch/epfl/sdp/blindly/splash_screen/SplashScreenActivityTest.kt

```diff
@@ -48,6 +51,15 @@ class SplashScreenActivityTest {
         release()
     }
 
+    @Test
+    fun splashScreenGoesToMainScreen() {
+        Mockito.`when`(user.isLoggedIn()).thenReturn(true)

```

Comment:
I think Tibor included this as default behavior in #251, if I'm not mistaken. Might shorten the test but not a big deal, you might as well leave it like this.

Reply:
I think we can leave this as-is. It will be easier to merge and it only isolate this test a bit, which I don't think is a problem.